demos/mac/cmac-aes256.c

```diff
@@ -66,23 +66,15 @@ static const char *propq = NULL;
 int main(void)
 {
     int rv = EXIT_FAILURE;
-    OSSL_LIB_CTX *library_context = NULL;
     EVP_MAC *mac = NULL;
     EVP_MAC_CTX *mctx = NULL;
     unsigned char *out = NULL;
     size_t out_len = 0;
     OSSL_PARAM params[4], *p = params;
     char cipher_name[] = "aes256";
 
-    library_context = OSSL_LIB_CTX_new();
-    if (library_context == NULL) {
-        fprintf(stderr, "OSSL_LIB_CTX_new() returned NULL\n");
-        goto end;
-    }
-    OSSL_LIB_CTX_set0_default(library_context);
```

Comment:
I think Shane meant to just omit this line.

Reply:
`EVP_MAC_init` is not able to find `AES256` without this line. :-(

Reply:
I'd call that a bug in CMAC ...

Reply:
It is found here when I try.

Reply:
It also works when I force the null provider into the default library context & use a separate lib ctx for the MAC operation.